CZI.md

```diff
@@ -204,32 +170,38 @@ contributors.
 Quantitatively evaluating maintenance work can be tricky---some Issues
 or PRs take minutes to review while others can take days to weeks of
 effort---but we believe that there is value at looking at the total
-number of open Issues and PRs.  We will reduce this number by closing
-Issues and PRs faster than they are opened until a reasonable
+number of open Issues and PRs. 
+<!--we wrote this in our progress report--> 
```

Comment:
It is worth repeating I think again here as this is what we will be writing next year's progress report against and it is still a true statement.